### DISCUSSION TITLE
Include group in disabled manager

### DISCUSSION THREAD

**Discussion by bilak** [renovatebot/renovate#17463] [Request Help]
Hi,
I'm using gradle with private repository (nexus) - maybe not relevant, just mentioning. 
I'd like to use renovate to be able to check for all our internal packages (e.g. `com.internal.something.*`) and then only for some provided groups (e.g. `group:springBoot`) but disable all others because spring-boot provides it's dependencies so it's not needed to upgrade everything what is provide by it. 

This should work when working with package patterns but is there a way how to include some groups but disable others? Or let's say only include some groups?

```javascript
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "local>eocs/tools/renovate-config//common-config",
    ":disableHost(repo.maven.apache.org)"
  ],
  "enabledManagers": ["gradle", "dockerfile", "gitlabci-include", "gitlabci"],
  "packageRules": [
    {
      "matchManagers": ["gradle"],
      "excludePackagePatterns": ["^com.internal.something.*", "^org.springframework.boot.*"],
      "enabled": false
    }
  ]
}

```

**Comment by PhilipAbed** [comment]
for every package individually we will go over all the package rules, and the last matching rule will apply to it.

we do not support for `MatchGroups` but that isn't a problem since every group is made using a certain package rule.
so you can use the same package rule to match a certain group.

so if a manager is enabled, you can exclude certain groups using package rules like you did above.

but if you disable a manager, then it is disabled from extraction, that means we will not search for files that match that package manager dependency files like `build.gradle` or others.
and Package Rules only apply for packages that we extract.
